### PR TITLE
feat(cellbasic): add icon support to CellBasic

### PR DESCRIPTION
**Pull request by masoudmanson**

### CODE REVIEW

File: src/core/CellBasic/index.stories.tsx [hunk 1]
```diff
@@ -25,12 +26,34 @@ const CellBasic = (props: Args): JSX.Element => {
...
+          "XS Download",
+          "S Light bulb",
+          "L Bacteria",
+          "XL Flask",
+        ],
+        type: "select",
+      },
+      mapping: availableIconOptions,
+      options: Object.keys(availableIconOptions),
+    },
```
Comment (bethbertozzi) [inline src/core/CellBasic/index.stories.tsx:56]: add icon as control to exclude in test

**Review by bethbertozzi — CHANGES_REQUESTED** [review]
I think it would be good to have a test with the icon, but other than that and a small cleanup it looks really great 😃 thanks Masoud!

File: src/core/CellBasic/index.test.tsx [hunk 2]
```diff
@@ -43,4 +43,11 @@ describe("<CellBasic />", () => {
     const style = window.getComputedStyle(renderedElement);
     expect(style.verticalAlign).toBe("bottom");
   });
+
```
Comment (bethbertozzi) [inline src/core/CellBasic/index.test.tsx:46]: 🙌

**Review by bethbertozzi — APPROVED** [review]
looks great!

**Review by clarsen-czi — APPROVED** [review]
LGTM! Thanks @masoudmanson !!

**Pull request merged**